src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -50,6 +52,12 @@ public virtual void Init(RequestHandlerContext context)
             _context = context;
         }
 
+        protected Stream RequestFormStream(string itemName)
```

Comment:
This doesn't belong here, you are trying to do something local in a shared space, and that isn't the place to do it.